db/migrate/20251126134647_add_updated_at_to_instrument_statuses.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class AddUpdatedAtToInstrumentStatuses < ActiveRecord::Migration[8.0]
+  def up
+    add_column :instrument_statuses, :updated_at, :datetime
+
+    # Set updated_at to created_at for existing records
+    execute "UPDATE instrument_statuses SET updated_at = created_at"
```

Comment:
What about setting `updated_at` after deleting the old records? That way we update way less records